recap.html

```diff
@@ -0,0 +1,184 @@
+    </head>
+    <body>
+        <header>
+            <div class="logo-content-box">
+                <div class="logo">
+                    <a href="index.html">
+                        <img src="resources/AI_Logo_Final.png"/>
+                        
+                    </a>
+                </div>
+            </div>
+            <div class="nav" align="center" role="navigation">
+                <div class="nav-button">
+                    <a href="index.html" role="button">
+                        About Us
+                    </a>
+                </div>
+                <div class="nav-button">
+                    <a href="meetings.html" role="button">
+                        Meetings
+                    </a>
+                </div>
+                <div class="nav-button">
+                    <a href="hackai.html" role="button">
+                        Hack AI
+                    </a>
+                </div>
+                <div class="nav-button">
+                    <a href="resources.html" role="button">
+                        Learning Resources
+                    </a>
+                </div>
+                <div class="nav-button">
+                    <a href="confs.html" role="button">
+                        AI conferences
+                    </a>
+                </div>
+            </div>
+        </header>
+        <div class="content" role="main">
+            <div class="content-box description">
+                <h2>
+                    Past HackAIs
+                </h2>
+
+                <img src="resources/Hack%20AI%20Logo.png" style="width:200px; padding-bottom: 20px"/>
+
+                <h3> <center> <u> HackAI 2022 Leaderboard </u> </center> </h3>     
+                <h5> <center> <a href="https://github.com/OSU-AI-Club/HackAI2022">2022 Project Repo</a> </center> </h5>
+                <br />
+                <br />
+
+                <h4> <center> <u> HackAI 2022 1st Place Winners (tied) </u> </center> </h4>
+
+
+                <p class="HackAI 2022 1st Place Winners">
+                    
+                    <b>Team Name:</b> Alvin and the Chipmunks <br />
+                    <br />
+                    <b>Members:</b> Mihir Joshi, Jagteshwar Singh, and Alvin Ishimwe <br />
+                    <br />
+                    <b>Project Name:</b> DermAI <br />
+                    <br />
+                    <b>Project Description:</b> "DermAI is an AI-based tool to assist physicians in making skin cancer diagnoses. At DermAI, our goal is to provide the highest accuracy prediction of common skin cancers to aid clinicians in making difficult decisions. Even if a clinician is not a dermatologist, they can use this app in their clinic and get an "extra set of eyes" on a potential cancer diagnosis. We believe this will be beneficial in healthcare settings as misdiagnosis is the leading cause of skin cancer-related deaths" - The DermAI Team<br />
```

Comment:
We can just take their description. I don't think we need quotes. Same for the other teams below